documentation/src/main/asciidoc/mapstruct-reference-guide.asciidoc

```diff
@@ -11,7 +11,7 @@
 This is the reference documentation of MapStruct, an annotation processor for generating type-safe, performant and dependency-free bean mapping code.
 This guide covers all the functionality provided by MapStruct. In case this guide doesn't answer all your questions just join the MapStruct https://groups.google.com/forum/?fromgroups#!forum/mapstruct-users[Google group] to get help.
 
-You found a typo or other error in this guide? Please let us know by opening an issue in the https://github.com/mapstruct/mapstruct[MapStruct GitHub repostory],
+You found a typo or other error in this guide? Please let us know by opening an issue in the https://github.com/mapstruct/mapstruct[MapStruct GitHub repository],
```

Comment:
Well, we could have said that it's there on purpose, just find out who the most vigilant readers are out there... 😉
Anyway, @xyzst: you won 🎉!

Seriously, Thanks a lot!